tools/iceoryx_build_test.sh

```diff
@@ -137,17 +158,19 @@ while (( "$#" )); do
         echo "Options:"
         echo "    -b --builddir         Specify a non-default build directory"
         echo "    -j --jobs             Specify the number of jobs to run simultaneously"
```

Comment:
I think we should get rid of this flag. We can acquire the number of cpu cores easily on a linux machine with:
```sh
cat /proc/cpuinfo | grep processor | wc -l
```

If we pack this into this shell script with 
```sh
NUMBER_OF_CORES=$(cat /proc/cpuinfo | grep processor | wc -l)
```

we speed up the script for most system since they all use the perfect number of cores.

Reply:
You also could consider using nproc